composer/tools/copy_environment.py

```diff
@@ -0,0 +1,761 @@
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START composer_copy_environment]
+"""Script to create a copy of an existing Cloud Composer Environment.
```

Comment:
Maybe add this documentation to a restructured text readme and generate the documentation using the <project-root>/scripts/readmegen.py script. [Example here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iot/api-client/manager/README.rst.in)